### PR TITLE
Add NuGet package for System.CommandLine

### PR DESCRIPTION
**Pull request by terrajobst**
@KrzysztofCwalina, please take a look.

@ahsonkhan, I've pushed System.CommandLine.0.1.0-d101215-1 to: https://www.myget.org/F/netcore-package-prototyping/api/v2/package.

### CODE REVIEW

File: nuget/System.CommandLine.nuspec [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    <owners>Microsoft</owners>
+    <licenseUrl>http://go.microsoft.com/fwlink/?LinkId=329770</licenseUrl>
+    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
+    <projectUrl>https://github.com/dotnet/corefxlab</projectUrl>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>CLI commandline command parsing</description>
+    <releaseNotes>Initial package</releaseNotes>
+    <copyright> Microsoft Corporation.  All rights reserved.</copyright>
+    <tags>Command line parsing support</tags>
+  </metadata>
```
Comment (KrzysztofCwalina) [inline nuget/System.CommandLine.nuspec]: No dependencies?

Comment (terrajobst) [inline nuget/System.CommandLine.nuspec]: Just followed what you did for your packages. I've honestly no idea how to build V3 packages yet. I can just add all the dependencies, but then I'm not sure what folder to put those in.

**Comment by KrzysztofCwalina** [comment]
Yeah, I need to update the other specs too, but I did add some dependencies to System.Text.Formatting.nuspec

**Comment by KrzysztofCwalina** [comment]
@terrajobst, are you merging it?

**Comment by KrzysztofCwalina** [comment]
Closed by mistake. reopening.

**Pull request merged**